osd/recovered_master_node.json

```diff
@@ -0,0 +1,8 @@
+{
+    "severity": "Error",
+    "service_name": "SREManualAction",
+    "cluster_uuid": "${CLUSTER_UUID}",
+    "summary": "SRE recovered master nodes",
+    "description" : "SRE have noticed that some master nodes of your cluster were stopped from the IaaS infrastructure. SRE have started back the nodes to recover the cluster. Please note that stopping master nodes manually is not a supported operation.",
```

Comment:
This should also include a case where a customer deletes the master node, so I suggest:

SRE have noticed a change in the state of one or more of the cluster's master nodes. SRE have restored the affected nodes. Please note that performing any action on the master nodes directly from the cloud provider manually is not supported.",

Reply:
+1. Though in some case we can't recover deleted master nodes, but if we can recover them, we can use this as notification.

Reply:
For "any action", I change it to "changing running state".  "Any action" sounds too wide.

Reply:
and it applies as well if they change it not from the cloud provider so you can remove that part of the sentence too
=> "Please note that if you change the running state of one of the master nodes manually then we won't be able to support your cluster anymore."